madminer/utils/morphing.py

```diff
@@ -810,4 +810,4 @@ def calculate_nuisance_factor_gradients(self, nuisance_parameters, benchmark_wei
         log_gradients = a + 2.0 * b * nuisance_parameters[:, np.newaxis]
         gradients = log_gradients * nuisance_factors[np.newaxis, :]
 
-        return gradients
+        return gradients
```

Comment:
This is a small detail, but please add this newline again so we're not touching this file in this pull request at all.